src/Tizen.NUI/src/public/WebView/WebContext.cs

```diff
@@ -86,63 +86,63 @@ internal WebContext(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cM
         /// <summary>
         /// Cache model
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
         public enum CacheModelType
         {
             /// <summary>
             /// The smallest cache capacity
             /// </summary>
-            [EditorBrowsable(EditorBrowsableState.Never)]
+            /// <since_tizen> 9 </since_tizen>
```

Comment:
we usually don't repeat the since_tizen for enum values if it matches the since_tizen of the whole enum, but it's not an error

Reply:
Done.